core/src/main/java/io/fabric8/maven/core/access/ClusterAccess.java

```diff
@@ -116,7 +116,7 @@ public boolean isOpenShift(Logger log) {
         } catch (KubernetesClientException exp) {
             Throwable cause = exp.getCause();
             String prefix = cause instanceof UnknownHostException ? "Unknown host " : "";
-            log.warn("Cannot access cluster for detecting mode: %s%s",
+            log.warn("Cannot access OpenShift cluster for detecting mode: %s%s",
```

Comment:
At this point, cluster can be either an OpenShift or kubernetes cluster. Why are you hardcoding Openshift?

Reply:
Oops I added by mistake. @theexplorist can you take care of this ?